scripts/data/get_block.py

```diff
@@ -65,6 +65,7 @@ def outpoint(prev_output):
                                     data: {tx_output({'vout': [prev_tx['vout'][prev_output['vout']]]})}
                                     block_height: {prev_block['height']}_u32,
                                     block_time: {prev_block['time']}_u32,
+                                    is_coinbase: {str(prev_output['vout'] == 0).lower()}
```

Comment:
It should be something like `prev_tx['vin'][0].get('coinbase') is not None`

Reply:
First transaction is not always the coinbase one?

Reply:
what is right way to know the coinbase transaction

Reply:
@maciejka  i was confuse on what do here, sow your last commit, everything is clear now